addons/binding/org.openhab.binding.feican/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="feican"
+                          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                          xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+                          xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <!-- Sample Thing Type -->
+    <thing-type id="bulb">
+        <label>Wifi RGBW Bulb</label>
+        <description>A dimmable light with changeable colors.</description>
+
+        <channels>
+            <channel id="switch" typeId="switch" />
```

Comment:
Sorry, but there's something wrong here - a thing type that has a color channel, must NOT have switch and brightness channels as those features are covered by the color channel already.